### PR TITLE
Commerce 9047

### PR DESCRIPTION
**Pull request by MahmoudAzzam**

### CODE REVIEW

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLinkLocalServiceImpl.java [hunk 1]
```diff
@@ -46,29 +46,29 @@
...
-			long cpDefinitionId1, long cpDefinitionId2, double priority,
-			String type, ServiceContext serviceContext)
+			long userId, long cpDefinitionId1, long cpDefinitionId2,
+			double priority, String type, ServiceContext serviceContext)
 		throws PortalException {
 
 		CPDefinition cpDefinition2 = cpDefinitionPersistence.findByPrimaryKey(
 			cpDefinitionId2);
 
 		return addCPDefinitionLinkByCProductId(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLinkLocalServiceImpl.java:56]: use service reference don't call method directly

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
1. Avoid direct method call, use service reference. I know this is something you didn't do, but if you find those mistakes please fix them
2. Don't change remote service signature
3. Pass userId to local service with method getUserId()

**Comment by riccardo-alberti** [comment]
@andrea-ale-sbarra is this PR still needed? If not please close

**Comment by andrea-ale-sbarra** [comment]
I'm closing this because the ticket was removed from the sprint. Once it come back we can review this PR as it is linked to the ticket

**Pull request closed without merging**